### PR TITLE
Go support

### PR DESCRIPTION
**Pull request by Fatalll**

### CODE REVIEW

File: plugin/build.gradle.kts [hunk 1]
```diff
@@ -7,7 +7,7 @@ version = rootProject.version
 
 intellij {
     pluginName = "Grazi"
-    version = Versions.intellij
+    version = "LATEST-EAP-SNAPSHOT"
```
Comment (TanVD) [inline plugin/build.gradle.kts:10]: Should fix a EAP

File: plugin/src/main/resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -58,6 +58,7 @@
     ]]>
     </change-notes>
 
+    <depends optional="true" config-file="go.xml">org.jetbrains.plugins.go</depends>
```
Comment (TanVD) [inline plugin/src/main/resources/META-INF/plugin.xml:61]: Let's check that everything works in GoLand

**Review by TanVD — CHANGES_REQUESTED** [review]

File: plugin/build.gradle.kts (hunk 1 above)
Comment (Fatalll) [reply plugin/build.gradle.kts:10]: Merge it after stable version released

File: plugin/src/main/resources/META-INF/plugin.xml (hunk 2 above)
Comment (Fatalll) [reply plugin/src/main/resources/META-INF/plugin.xml:61]: Everything works

**Pull request closed without merging**